### ISSUE TITLE
Unable to run the app on Firefox

### ISSUE THREAD

**Issue by Raj123456788** [emannion/webrtc-screen-share#1]
Hello There,
I followed the instructions for chrome but still unable to run the app on Firefox. It says "screen capturing is either denied or not supported". How can I troubleshoot this? also  my node server is accepts the connection. But I get following errors:
1) "e no defined" error.

2) The operation is insecure.
I am waiting for the reply.

**Comment by viviemXD** [comment]
Did you setup these Firefox options in   about:config

(Only for Firefox) open about:config create media.getusermedia.screensharing.enabled and set to true
(Only for Firefox) open about:config in media.getusermedia.screensharing.allowed_domains append the IP address of your node server e.g. "192.168.1.2"
Point two browsers to, e.g.    **https**://<your node.js ip address>

Accept any local unsecure certificate dialogues that appear on either browser.

**Comment by Raj123456788** [comment]
yes, i did. i have your chrome app(https://github.com/emannion/webrtc-application-screen-share) packaged app working but I am unable to run it on Firefox? I get above error I can see it getting connected. It gives me securityError. The operation is unsure. I am using ws and not wss. Can we make it work?

**Comment by Raj123456788** [comment]
I am waiting for your reply Sir!

**Comment by viviemXD** [comment]
Please tell me you are not trying to run a chrome packaged app oon Firefox.  It is not clear what you are trying to do. 

Sorry I do not check these messages very often.
